proto/panacea/datadeal/v2/consent.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
```

Comment:
I don't know this file name is correct.
certificate.go? consent.go? 
Please give me a good idea!

Reply:
OK. I changed name to `certificate.proto`.
Thank you